core/domain/config_domain.py

```diff
@@ -408,10 +408,6 @@ def get_all_config_property_names(cls):
     'Always ask learners for answer details. For testing -- do not use',
     False)
 
-CLASSROOM_PAGE_IS_ACCESSIBLE = ConfigProperty(
```

Comment:
Make sure to add it to the list of deprecated properties (line 170).

Reply:
done